core/rawdb/accessors_chain.go

```diff
@@ -864,13 +864,13 @@ func ReadAllBadBlocks(db ethdb.Reader) []*types.Block {
 	if err := rlp.DecodeBytes(blob, &badBlocks); err != nil {
 		return nil
 	}
-	var blocks []*types.Block
-	for _, bad := range badBlocks {
+	blocks := make([]*types.Block, len(badBlocks))
+	for i, bad := range badBlocks {
 		block := types.NewBlockWithHeader(bad.Header)
 		if bad.Body != nil {
 			block = block.WithBody(*bad.Body)
 		}
-		blocks = append(blocks, block)
+		blocks[i] = block
```

Comment:
Please revert this. This is run once in a blue moon, it's not epformance critical to make it less readable.

Reply:
Done